### DISCUSSION TITLE
Unable to Save Billing Details for GitHub Enterprise Subscription - Missing GST ID

### DISCUSSION THREAD

**Discussion by kishan1219** [community/community#141415] [Enterprise] [answered]
### Select Topic Area

Question

### Body

Hello GitHub Community,

I am seeking assistance with an issue regarding the redemption of a GitHub Enterprise coupon for my organization, VidyaInterna Hub. While attempting to redeem the coupon, I encountered a roadblock in the billing process as I am unable to provide a GST ID, which is a required field for businesses in India.

My company is a small edtech startup in India that is currently not registered for GST, as we fall below the threshold for mandatory GST registration. Despite this, we would like to proceed with the GitHub Enterprise subscription using the coupon we’ve received. Here are my details:

Organization Name: VidyaInterna Hub

Coupon Applied: $452.00/year coupon for GitHub Enterprise Plan
Could you please guide me on how to proceed with redeeming the coupon without a GST ID? Is there an alternative method to complete the billing details or any option to bypass this requirement for startups like ours?

Thank you for your assistance, and I look forward to your guidance on resolving this issue.

Best regards,
Kishan Gupta
Founder, VidyaInterna Hub
Email: Kishan@VidyaInternaHub.onmicrosoft.com

**Comment by queenofcorgis** [comment, marked as answer]
Hi @kishan1219 ! I appreciate your patience. The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
